bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp

```diff
@@ -610,8 +613,8 @@ DataGenerator::DataGenerator(const Config& targetConfig, const TestConfig& testC
  */
 void DataGenerator::generateBlocks(TestAppState& appState, uint64_t fromBlockId, uint64_t toBlockId) {
   std::unique_ptr<char[]> buff = std::make_unique<char[]>(Block::getMaxTotalBlockSize());
-  ConcordAssertEQ(toBlockId % testConfig_.checkpointWindowSize, 0);
-  ConcordAssertGT(fromBlockId, 1);
+  // ConcordAssertEQ(toBlockId % testConfig_.checkpointWindowSize, 0);
```

Comment:
nit: Is this comment required?

Reply:
I should completely remove it, we are not restricted to this condition..
Thanks.